plugins/vpc/multipleSubnets.js

```diff
@@ -22,14 +24,10 @@ module.exports = {
 			LocalAWSConfig.region = region;
 			var ec2 = new AWS.EC2(LocalAWSConfig);
 
-			var params = {
-
-			};
-
-			ec2.describeVpcs(params, function(err, data){
-				if (includeSource) source[region] = {error: err, data: data};
+			helpers.cache(cache, ec2, 'describeVpcs', function(err, data) {
```

Comment:
Using the local cache ensures we only call `describeVpcs` once per region across all plugins.